README.md

```diff
@@ -227,11 +227,17 @@ The filename of a definition file will determine the name of the image repositor
 
 The manifest file format is officially based on [RFC 2822](https://www.ietf.org/rfc/rfc2822.txt), and as such should be familiar to folks who are already familiar with the "headers" of many popular internet protocols/formats such as HTTP or email.
 
+The primary additions are inspired by the way Debian commonly uses 2822 -- namely, lines starting with `#` are ignored and "paragraphs" (or "entries") are separated by a blank line.
+
+The first entry is the "global" metadata for the image. The only required field in the global entry is `Maintainers`, whose value is comma-separated in the format of `Name <email> (@github)` or `Name (@github)`. If fields such as `GitRepo` are specified in the global entry, they will be the default for the remainder of the file.
```

Comment:
> If fields such as `GitRepo`

It feels like this might make people think that only `GitRepo` is valid in the global section as a default.  Maybe something like this? 
> Any field specified in the global entry will be the default for the rest of the entries/paragraphs and can be overridden in an individual paragraph